pkg/utils/miniojob/minioJob.go

```diff
@@ -176,6 +181,58 @@ func parseK8SFile(out *Arg, key, ext, val string) error {
 	return nil
 }
 
+// readFromK8s - support set with get(secret|configmap.namespace.name.key)
+func readFromK8s(val *string) error {
+	if !strings.HasPrefix(*val, "get(") {
+		return nil
+	}
+	fromInfos := strings.SplitN(strings.TrimPrefix(strings.TrimSuffix(*val, ")"), "get("), ".", 4)
```

Comment:
Wait we have to write it like this and no other standardized way is available to support this?

Reply:
@jiuker I unresolved this conversation, because I just added a comment while it was resolved.

Reply:
@ramondeklein Now namespace hidden. Please review again.

Reply:
Removing the namespace is good, but I think we should investigate/discuss whether or not we can use the syntax that is used for environment variables. When I saw it yesterday, I thought about that syntax. @harshavardhana also has the same feeling, so I think we should postpone this until we decide what to do with that.